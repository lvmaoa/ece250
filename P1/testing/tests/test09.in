m 7
push_front "news" "news.google.com"
push_back "google" "www.google.com"
push_front "stackexchange" "www.stackexchange.com"
clear
pop_front
print
size
pop_back
print
size
push_back "bored" "www.boredpanda.com"
print
size
push_back "uw" "www.uwaterloo.ca"
push_front "piazza" "www.piazza.com"
print
pop_front
size
clear
print
exit

