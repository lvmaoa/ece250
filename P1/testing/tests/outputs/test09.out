success
success
success
success
success
failure
deque is empty
size is 0
failure
deque is empty
size is 0
success
"bored" "www.boredpanda.com"
size is 1
success
success
"uw" "www.uwaterloo.ca"
"bored" "www.boredpanda.com"
"piazza" "www.piazza.com"
success
size is 2
success
deque is empty
